src/lxc/conf.c

```diff
@@ -3351,6 +3351,8 @@ int lxc_assign_network(const char *lxcpath, char *lxcname,
 		netdev = iterator->elem;
 
 		if (netdev->type == LXC_NET_VETH && !am_root) {
+			if (netdev->mtu)
+				WARN("ignoring MTU due to unprivileged");
```

Comment:
Could you make this an `INFO()` and reword it to: "mtu ignored due to insufficient privilege"? That would be awesome.